pkg/collector/corechecks/cluster/kubernetesapiserver/kubernetes_apiserver.go

```diff
@@ -42,6 +42,7 @@ const (
 
 	defaultCacheExpire = 2 * time.Minute
 	defaultCachePurge  = 10 * time.Minute
+	defaultTimeout     = time.Second
```

Comment:
I think 1 second can be too short, we have a config `kubernetes_apiserver_client_timeout` defaulted to `10` already, should we use it?

Reply:
Huh, looks like `kubernetes_apiserver_client_timeout ` already the global default. It seems better to remove the context timeout entirely then 👍